test/users.js

```diff
@@ -199,59 +199,10 @@ describe("User cycle", async () => {
       assert.equal(responseData.response.error, undefined);
     });
 
-    it("should fail incorrect login", async () => {
-      const requestData = {
-        request: {
-          user_edit: {
```

Comment:
can we still have a success test for `user_edit ` where we edit full_name, login, email, phone ?